src/applications/vaos/referral-appointments/CompleteReferral.jsx

```diff
@@ -0,0 +1,254 @@
+  const { search } = useLocation();
+  const history = useHistory();
+  const dispatch = useDispatch();
+  useEffect(
+    () => {
+      dispatch(setFormCurrentPage('complete'));
+    },
+    [dispatch],
+  );
+
+  const basePath = useRouteMatch();
+  const { provider, loading, failed } = useGetProviderById(
+    currentReferral?.providerId,
+  );
+
+  if (failed) {
+    return (
+      <ReferralLayout>
+        <CCAppointmentCard>
+          <va-alert status="error" data-testid="error-alert">
+            <p className="vads-u-margin-y--0">
+              We’re sorry. There was a problem with our system. We couldn’t
+              process this appointment. Call us at 877-470-5947. Monday through
+              Friday, 8:00 a.m. to 8:00 p.m. ET.
+            </p>
+          </va-alert>
+        </CCAppointmentCard>
+      </ReferralLayout>
+    );
+  }
+
+  const savedSelectedSlotKey = getReferralSlotKey(currentReferral.UUID);
+  const savedSlotId = sessionStorage.getItem(savedSelectedSlotKey);
+
+  if (loading && !failed) {
+    return (
+      <div className="vads-u-margin-y--8" data-testid="loading">
+        <va-loading-indicator message="Loading your appointment details" />
+      </div>
+    );
+  }
+
+  if (!savedSlotId || !provider) {
+    return <Redirect from={basePath.url} to="/" />;
+  }
+
+  const params = new URLSearchParams(search);
+  const comfirmMessage = params.get('confirmMsg') === 'true';
+
+  const savedSlot = getSlotById(provider.slots, savedSlotId);
+
+  const savedDate = new Date(savedSlot.start);
+  const timeZoneAbr = getTimezoneAbbrByFacilityId(
+    currentReferral.ReferringFacilityInfo.FacilityCode,
+  );
+  const timeZone = getTimezoneByFacilityId(
+    currentReferral.ReferringFacilityInfo.FacilityCode,
+  );
+
+  const appointmentDuration = intervalToDuration({
+    start: new Date(savedSlot.start),
+    end: new Date(savedSlot.end),
+  });
+
+  return (
+    <ReferralLayout>
+      <CCAppointmentCard>
+        <div
+          data-testid="referral-content"
+          className="vads-u-background-color--success-lighter  vads-u-padding-x--2 vads-u-padding-y--1 vads-u-margin-y--2p5"
+        >
+          <h3 className="vads-u-margin-top--0 vads-u-margin-bottom--2p5 vads-u-font-size--md">
+            {comfirmMessage
+              ? 'We’ve scheduled and confirmed your appointment.'
+              : 'You already scheduled your first appointment for this referral'}
+          </h3>
+          {comfirmMessage && (
+            <>
+              <va-link
+                href="#"
+                data-testid="review-appointments-link"
+                onClick={e => {
+                  e.preventDefault();
+                  routeToCCPage(history, 'appointments');
+                }}
+                text="Review your appointments"
+              />
+              <div className="vads-u-margin-top--2">
+                {/* TODO: find where this links to and add link to schedule a new appointment */}
```

Comment:
Think this might go away and is maybe something left over from the new appointment flow. We should have a quick chat with UX around several things in the complete page.